src/Data/Refinement/Properties.agda

```diff
@@ -0,0 +1,23 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Properties of refinement types
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+module Data.Refinement.Properties where
+
+open import Level
+open import Data.Refinement
+open import Relation.Binary.PropositionalEquality.Core using (_≡_; refl)
+
+private
+  variable
+    a p : Level
+    A : Set a
+
+module _ {P : A → Set p} where
```

Comment:
As `P` is used in the type of `Refinement`, it's fine to add it as a variable. In fact you could do this in `Data.Refinement` as well if you were feeling keen.

Reply:
Will do!